### PR TITLE
arch: soc: nrf52: add UART1 to dts fixup

### PR DESCRIPTION
**Pull request by barsok**
Current fixup supports only UART0 while there is also UART1

Signed-off-by: Bartosz Sokolski <bartosz.sokolski@nordicsemi.no>

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]

File: arch/arm/soc/nordic_nrf/nrf52/dts.fixup [hunk 1]
```diff
@@ -5,6 +5,10 @@
 #define CONFIG_UART_0_BAUD_RATE		NORDIC_NRF_UART_40002000_CURRENT_SPEED
 #define CONFIG_UART_0_NAME		NORDIC_NRF_UART_40002000_LABEL
 
+#define CONFIG_UART_1_IRQ_PRI		NORDIC_NRF_UART_40028000_IRQ_0_PRIORITY
+#define CONFIG_UART_1_BAUD_RATE		NORDIC_NRF_UART_40028000_CURRENT_SPEED
```
Comment (carlescufi) [inline arch/arm/soc/nordic_nrf/nrf52/dts.fixup]: Is it GitHub or is the alignment wrong here?

Comment (ioannisg) [reply arch/arm/soc/nordic_nrf/nrf52/dts.fixup]: Do we need to speify that this is nRF52840 only?

**Review by ioannisg — CHANGES_REQUESTED** [review]
LGTM; left a quick issue to clarify, only.

File: arch/arm/soc/nordic_nrf/nrf52/dts.fixup (hunk 1 above)
Comment (barsok) [reply arch/arm/soc/nordic_nrf/nrf52/dts.fixup]: Yes. of all nrf51/52 only nrf52 has the second uart so there is no need to update the fixup for nRF51. By the time nRF91 is merged I hope we will have decided to use defines from DT directly; besides it is not there in the main repo anyway

Comment (barsok) [reply arch/arm/soc/nordic_nrf/nrf52/dts.fixup]: Yes, as only nrf52 has two uarts

Comment (ioannisg) [reply arch/arm/soc/nordic_nrf/nrf52/dts.fixup]: Discussed offline with @barsok 
Proposed to leave this PR as is and take the discussion of whether to have conditional defines in fixups offline.

**Review by ioannisg — APPROVED** [review]

**Pull request merged**